examples/pytorch_to_stockroom_loader.py

```diff
@@ -0,0 +1,130 @@
+import argparse
+from pathlib import Path
+from typing import Union
+
+import torch
+from torch.utils.data import Dataset
+from torchvision import datasets, transforms
+from stockroom import StockRoom, init_repo
+from hangar import Repository
+import numpy as np
+from tqdm import tqdm
+
+from dataset_configs import dataset_configs
+
+
+def hangar_transform(item):
+    if isinstance(item, torch.Tensor):
+        return item.detach().numpy()
+    if isinstance(item, int):
+        return np.array([item])
+    if isinstance(item, list):
+        subsample = dict()
+        for idx, i in enumerate(item):
+            subsample[idx] = i
+        return subsample
+
+
+def add_from_dataset(
+        name: str,
+        dataset: Dataset,
+        root_dir: Union[Path, str],
+        column_postfixes: list = None
+        ) -> str:
```

Comment:
This is a pretty massive function. Maybe split it up into more "digestable" components?

Reply:
Also, @hhsecond please review the interaction between what occurs between the `Hangar` API and `Stockroom` API. After all the columns are created, we'll definitely want to use the hangar `bulk_import` utility, but I'd like to be sure that it's going to play nice with how stockroom deals with prepending identifiers onto keys / sentinel values.

Reply:
Yep, that is something I should do I guess but I'm currently working on adding the important datasets from torchtext and torchaudio. This will make me familiar a variety of datasets and after that, I'll be able to do a more thorough refactoring. 

Also like you mentioned we should try to build from more general data sources first and those functionalities should be added into hangar. I'm trying to think of a more generalizable way to do the whole thing and hopefully, after completing text and audio we will have a better idea on how that should be done I guess.

Reply:
Great! We'll put this refactor on the backburner for now